lib/awful/widget/layoutbox.lua

```diff
@@ -60,7 +59,7 @@ function layoutbox.new(screen)
         w._layoutbox_tooltip = tooltip({ objects = {w}, delay_show = 1 })
 
         update(w, screen)
-        boxes[screen] = w
+        boxes[screen] = setmetatable({}, {__index = w, __newindex = w})
```

Comment:
How does wrapping the widget in an empty table change anything? This new entry keeps the widget alive, so just using `= w` here should have the same effect.

But whatever, if it works, I'll not question it.